src/test/java/uk/gov/hmcts/reform/civil/model/citizenui/DashboardClaimStatusFactoryTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import uk.gov.hmcts.reform.civil.enums.CaseState;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+class DashboardClaimStatusFactoryTest {
+
+    private final DashboardClaimStatusFactory factory = new DashboardClaimStatusFactory();
+
+    @Test
+    void decisionOutcomeCcdClaimant() {
```

Comment:
Could you rename this test? As it will conflict with the names in the other civil-service PR and does not reflect content.